packages/react-native.config.js

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Remove?

Reply:
👍 Should this have ended up inside `packages/expo-random` to add RN autolinking capabilities?

Reply:
ah yes